src/book.py

```diff
@@ -56,23 +56,23 @@ def create_operation(
         coin: str,
         row: int,
         file_path: Path,
-    ) -> Optional[tr.Operation]:
+    ) -> Type[tr.Operation]:
```

Comment:
This function does not return a type of operation but an object operation. The typing.type is too much.

Reply:
Then I'm getting
`src/book.py:74:9: error: Returning Any from function declared to return "Operation"  [no-any-return]`
even when asserting after `getattr`:
`assert issubclass(Op, tr.Operation)`

Reply:
Nevermind, works now, `assert isinstance(op, tr.Operation)` has to be done directly before return